components/05-page-examples/standard-page.twig

```diff
@@ -9,4 +9,24 @@
     {% include "@molecules/text/text-field.twig" with {
       text_field__content: '<p>The undergraduate program in Chemistry at Yale reflects the position of chemistry as the foundation of all the molecular sciences. Students are equipped with the technical skills to appreciate the scientific basis for previous discoveries and develop the fundamental expertise required to make future breakthroughs. Under the tutelage of world-leading researchers, students are exposed to a broad range of topics in chemistry. The development of technical skills through lecture classes is complemented with hands-on experience in state-of-the-art chemistry laboratories.</p><p>Many students also perform independent laboratory research under the guidance of a faculty mentor. This rigorous training prepares students for professional careers in a diverse array of fields by teaching them how to apply the scientific method, providing them with skills in quantitative reasoning, and exposing them to scientific research.</p><p>After graduation, students with a B.A. or B.S. degree often pursue work or further studies in chemistry, biochemistry, or health-related disciplines, but also find their broad scientific training beneficial in energy research, policy, environment, business management, and law. As the problems of contemporary society involve ever more complex scientific issues, degree programs in the sciences become increasingly appropriate for students wishing to pursue careers in public policy, government, or public service.</p>',
     } %}
+    {% include "@atoms/divider/divider.twig" %}
+    {% include "@molecules/text/text-field.twig" with {
+      text_field__content: '<h2>Program Inforamtion</h2><h3>The Undergraduate Handbook</h3><p>A comprehensive guide for prospective and current Chemistry Majors, with a complete description of requirements and opportunities.</p><p><a href="#">View the handbook online</a></p><h3>Course List</h3><p>A list of courses and a description of the Chemistry undergraduate program, including information on placement exams, laboratory registration, premedical students, and major requirements</p><p><a href="#">Yale College programs of study</a></p>',
```

Comment:
Not a blocker for this PR but we should probably move all this extra twig/storybook data to it's own file. As the project progresses these static values are really going to muddy up this file.

Reply:
We talked about the position of the button, and decided to keep it up "next to" the content instead of pushing it down. There are various screen sizes and content lengths that affect the amount of space that is created, and it was decided that keeping it close was preferred.

And about keeping content out of the page-examples... Evan always said the same thing, but I prefer to keep it in the twig files for example pages. We can talk it through if you ever want to, but I've found that when there are multiples of the same component on a page (like two text fields) you have to namespace them (`text__content: text__content__1` and `text__content: text__content__2` etc.) in order to get the right content in the right components. Just putting it in the file means you don't have to do that extra mapping... and you don't get a cluttered folder (two files per page instead of one). I think this is a personal preference thing, and I'm going to keep it like it is for now, but if it does ever seem like it's getting overwhelming, it'll be an easy refactor.

Reply:
Sounds good regarding the design. Figured I would mention it.

hmmm Evan was correct. RIP.